README.md

```diff
@@ -13,6 +13,19 @@ Takeaway Challenge
        ':..:'                ':..:'
 
  ```
+Marianne October 2019 Cohort -
+
+1 test failing - attempt to prevent text from sending each time rspec is run.
+Also currently am able to send texts to my own number but have no corresponding rspec tests to confirm this.
+Need to deal with case where user adds something which is not on the menu.
+
+To use:
+* Load takeaway.rb in IRB
+* takeaway = Takeaway.new
+* takeaway.start_order
+* takeaway.new_order.add("Margherita")
```

Comment:
Thinking further ahead in the life of the program, I'm not sure of the wisdom of storing the last order on the Takeaway/Restaurant, and then recommending people interact with that. What if two people are ordering at the same time? When the second person starts their order, it will replace the previous unfinished `@new_order`. They will both be adding items to whichever order was most recently created.

Reply:
Just noticed that the above doesn't apply because you are creating a new Takeaway for each user. I made that assumption because of how in real life there will only be one restaurant, which centrally manages what is contained in the menu, and can take orders, etc.